modules/nf-core/biscuit/bsconv/meta.yml

```diff
@@ -1,52 +1,48 @@
-name: biscuit_bsconv
-description: Summarize and/or filter reads based on bisulfite conversion rate
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "biscuit_bsconv"
+description: write your description here
 keywords:
-  - biscuit
-  - DNA methylation
-  - WGBS
-  - scWGBS
-  - bisulfite sequencing
-  - aligner
-  - bam
-  - filter
+  - sort
+  - example
+  - genomics
```

Comment:
looks like this is a regression. Keywords and description should accurately reflect the tool